libs/ui/src/unconfigured_election_screen.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { ConfigurationError } from '@votingworks/types';
+import { throwIllegalValue } from '@votingworks/basics';
+import { UsbDriveStatus } from './hooks/use_usb_drive';
+import { IndeterminateProgressBar } from './graphics';
+import { CenteredLargeProse } from './centered_large_prose';
+
+interface Props {
+  usbDriveStatus: UsbDriveStatus;
+  backendConfigError?: ConfigurationError;
+}
+
+export function UnconfiguredElectionScreen({
+  usbDriveStatus,
+  backendConfigError,
+}: Props): JSX.Element {
+  const errorMessage = (() => {
+    if (usbDriveStatus !== 'mounted') {
+      return 'Insert a USB drive containing a ballot package.';
+    }
+
+    if (!backendConfigError) {
+      return undefined;
+    }
+    switch (backendConfigError) {
+      case 'no_ballot_package_on_usb_drive':
+        return 'No ballot package found on the inserted USB drive.';
+      /* istanbul ignore next - compile time check for completeness */
+      default:
+        throwIllegalValue(backendConfigError);
+    }
+  })();
+
+  if (errorMessage) {
+    return (
+      <CenteredLargeProse>
+        <h1>VxScan is not configured</h1>
+        <p>{errorMessage}</p>
+      </CenteredLargeProse>
+    );
+  }
+
+  return (
+    <CenteredLargeProse>
+      <h1>Configuring VxScan from USB drive…</h1>
+      <IndeterminateProgressBar />
+    </CenteredLargeProse>
```

Comment:
This has moved to `libs/ui`, but this copy is still very VxScan specific. Will it be parameterized in a later change? Otherwise why have it in `libs/ui`?

Reply:
Yes exactly, will be parameterized since I'm not exactly sure what needs to be parameterized (eg. passing just machine name vs passing a whole string)